efel/api.py

```diff
@@ -33,6 +33,8 @@
 import efel
 import efel.cppcore as cppcore
 
+import efel.pyfeatures as pyfeatures
```

Comment:
from efel import cppcore, pyfeatures ?

Reply:
Yeah, Werner implemented some new features just in Python. One issue I found in using Python-only features is that getDistance function knows only about c++ features names. 

(https://github.com/BlueBrain/eFEL/blob/master/efel/api.py#L196)

I can try to think about a work-around.